src/app/modules/form/admin-form/admin-form.routes.ts

```diff
@@ -0,0 +1,40 @@
+import { celebrate, Joi, Segments } from 'celebrate'
+import { Router } from 'express'
+
+import { SettingsUpdateDto } from '../../../../shared/typings/form'
+import { AuthType, Status } from '../../../../types'
+import { withUserAuthentication } from '../../auth/auth.middlewares'
+
+import { handleUpdateSettings } from './admin-form.controller'
+
+export const AdminRouter = Router()
+
+// All routes in this handler should be protected by authentication.
+AdminRouter.use(withUserAuthentication)
+
+/**
+ * Joi validator for PATCH /form/:formId/settings route.
+ */
+const updateSettingsValidator = celebrate({
+  [Segments.BODY]: Joi.object<SettingsUpdateDto>({
+    authType: Joi.string().valid(...Object.values(AuthType)),
+    emails: Joi.alternatives().try(
+      Joi.array().items(Joi.string()),
+      Joi.string(),
+    ),
+    esrvcId: Joi.string().allow(''),
+    hasCaptcha: Joi.boolean(),
+    inactiveMessage: Joi.string(),
+    status: Joi.string().valid(...Object.values(Status)),
+    submissionLimit: Joi.number().allow(null),
+    title: Joi.string(),
+    webhook: Joi.object({
```

Comment:
permissionList is a collaborator thing, not settings right

Reply:
hmm I think they are separate in the UI, but semantically I'd say the permissionList is part of the settings, right? the alternative is to create another `/collaborators` endpoint which will look nearly identical to this one

Reply:
Should we not? Even though it may be nearly identical, at least that endpoint only does a single thing and gates properly? Can have PUT collaborators and GET collaborators hahaha

Reply:
As discussed during syncup, decision is to keep `/collaborators` as a separate endpoint